internal/settings/settings.go

```diff
@@ -259,7 +259,6 @@ type OutputPath struct {
 // It is meant to be set by a YAML or JSON config file, or flags.
 type ExternalConfig struct {
 	Excludes           []string `json:"excludes,omitempty" yaml:"excludes,omitempty"`
-	NoDefaultExcludes  bool     `json:"no_default_excludes,omitempty" yaml:"no_default_excludes,omitempty"`
```

Comment:
this should get a changelog entry

Reply:
It already has one https://github.com/uber/prototool/blob/dev/CHANGELOG.md